test/verification/test_entanglemet.py

```diff
@@ -0,0 +1,75 @@
+# -*- coding: utf-8 -*-
+
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2020.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+# pylint: disable=no-name-in-module
+
+"""
+Test entanglement functions of Ignis verification
+"""
+
+import unittest
+
+import numpy as np
+
+from qiskit.ignis.verification.entanglement.linear import *
```

Comment:
You only test linear, not parallel or analysis, which are the "heavier" files here.

Reply:
Yes, will get to it now, as all others in actual code are done